build/azure-pipelines/build-common.yml

```diff
@@ -49,7 +49,7 @@ steps:
   inputs:
     command: 'build'
     arguments: '-o $(Build.BinariesDirectory) -ldflags="-X main.version=${{ parameters.VersionTag }}"'
-    workingDirectory: '$(Build.SourcesDirectory)/cmd/sqlcmd'
+    workingDirectory: '$(Build.SourcesDirectory)/cmd/modern'
```

Comment:
don't we have to pass the binary name too? Otherwise it's going to call it "modern.exe"

Reply:
Added binary name via variable matrix